README.md

```diff
@@ -46,6 +46,7 @@ This template comes with:
 - [Factory Bot](https://github.com/thoughtbot/factory_bot) for testing data
 - [Faker](https://github.com/stympy/faker) for generating test data
 - [Figaro](https://github.com/laserlemon/figaro) for handling environment variables
+- [Omniauth Google Oauth2](https://github.com/zquestz/omniauth-google-oauth2) for Google Sign Up/Sign in
```

Comment:
I think this should go below `Letter Opener` because it's alphabetically sorted

Reply:
lol that's what I was trying to do, fail